integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/repository/CoinBaseRepository.kt

```diff
@@ -147,7 +147,7 @@ interface CoinBaseRepositoryInt {
     suspend fun getActivePaymentMethods(): ResponseResource<List<PaymentMethodsData>>
     suspend fun placeBuyOrder(placeBuyOrderParams: PlaceBuyOrderParams): ResponseResource<PlaceBuyOrderUIModel>
     suspend fun commitBuyOrder(buyOrderId: String): ResponseResource<CommitBuyOrderUIModel>
-    suspend fun sendFundsToWallet(sendTransactionToWalletParams: SendTransactionToWalletParams): ResponseResource<Int>
+    suspend fun sendFundsToWallet(sendTransactionToWalletParams: SendTransactionToWalletParams): ResponseResource<Response<SendTransactionToWalletResponse?>>
```

Comment:
The response type of this method seems somewhat convoluted. There are 3 nested "Responses" of some kind.

When you have a suspend method, there isn't really a reason to use a class like `ResponseResource` in my opinion. The exception is propagated, so you have response/error functionality already, just need to try/catch it. `Loading` object isn't used in Coinbase right now, and the loading state can be implemented in other ways.